frontend-vue-js/src/util/dateHandler.js

```diff
@@ -0,0 +1,3 @@
+export function getStringDateFromMilliseconds(dateInMilliseconds) {
+    return new Date(dateInMilliseconds).toLocaleDateString("en-UE");
+}
```

Comment:
I add a new util method that converts milliseconds to string. It's very useful because we often need to make this type of conversion.